l10n_es_aeat_silicie/models/stock_move.py

```diff
@@ -0,0 +1,683 @@
+                    move = self.env["stock.move"].browse(
+                        int(datos["NumReferenciaInterno"]))
+                    if move:
+                        move.silicie_move_number = datos["NumeroAsiento"]
+                        move.silicie_entry_type = datos["TipoAsiento"]
+                        move.date_send_silicie = fields.datetime.now()
+            if res["ErrorSalida"]:
+                raise exceptions.Warning(res)
+
+    @api.multi
+    def get_test_mode(self, port_name):
+        return port_name
+
+    @api.multi
+    def _get_data_dict(self, lot):
+        self.ensure_one()
+        container_code = self.container_type_silicie_id.code or ""
+        factor_conversion = self.factor_conversion_silicie
+        qty_done = round(self.quantity_done)
+        if not container_code:
+            factor_conversion = ""
+            qty_done = ""
+        return {
+            "partner_name": self.picking_id.partner_id.name or "",
+            "alcoholic_grade": self.alcoholic_grade or "",
+            "absolute_alcohol": self.absolute_alcohol or "",
+            "container_code": container_code,
+            "factor_conversion": factor_conversion,
+            "qty_done": qty_done}
+
+    @api.multi
+    def _prepare_values(self, lot):
+        data = self._get_data_dict(lot)
+        return {
+            "Número Referencia Interno": self.id,
+            "Número Asiento Previo": "",
+            "Fecha Movimiento": self.date.strftime("%d/%m/%Y"),
+            "Fecha Registro Contable": self.date.strftime("%d/%m/%Y"),
+            "Tipo Movimiento": self.silicie_move_type_id.code,
+            "Información adicional Diferencia en Menos":
+                self.silicie_loss_id.code if self.silicie_loss_id else "",
+            "Régimen Fiscal": self.silice_tax_position,
+            "Tipo de Operación":
+                self.silicie_processing_id.code if
+                self.silicie_processing_id else "",
+            "Número Operación":
+                self.silicie_operation_num if
+                self.silicie_operation_num else "",
+            "Descripción Unidad de Fabricación": "",
+            "Código Unidad de Fabricación": "",
+            "Tipo Justificante": self.silicie_proof_type_id.code,
+            "Número Justificante": self.reference,
+            "Tipo Documento Identificativo": "1",
+            "Número Documento Identificativo":
+                self.picking_id.partner_id.vat or
+                self.company_id.vat or "",
+            "Razón Social": data["partner_name"][:125],
+            "CAE/Número Seed": "",
+            "Repercusión Tipo Documento Identificativo": "",
+            "Repercusión Número Documento Identificativo": "",
+            "Repercusión Razón Social": "",
+            "Epígrafe": self.epigraph_silicie_id.fiscal_epigraph_silicie or "",
+            "Código Epígrafe": self.epigraph_silicie_id.code or "",
+            "Código NC": self.nc_code,
```

Comment:
Lo mismo que arriba, longitud <=8:
`"Código NC": self.nc_code and self.nc_code[:8],`